app.js

```diff
@@ -167,26 +91,15 @@ app.get('/healthcheck', function(req, res) {
   });
 });
 
-// Redirect / to a place on webmaker.org (namely webmaker.org/goggles),
-// since corresponding activities and learning materials are there.
-var webmakerGogglesLanding = env.get("audience") + "/goggles";
-app.get("/", function(req, res) {
-  res.redirect(301, webmakerGogglesLanding);
-});
-
-// Redirect this route to "/" to be safe if anyone is still using it.
-// 2014/02/08 we have removed route to /index.html in this #Bug973991
-app.get("/index.html", function(req, res) {
-  res.redirect(301, "/");
+// oauth login confirmation page
+app.get("/login-confirmation.html", function(req, res) {
+  res.render("login-confirmation.html");
```

Comment:
we could just move this into the public dir, too, since it doesn't need any template substitutions. It's just more convenient to have it live in `views`, so that if you need to work on any of the files, they're in the same dir